test/e2e/main/main_test.go

```diff
@@ -817,3 +818,73 @@ func throwHTTPError(resp *http.Response) {
 	errorBytes, _ := ioutil.ReadAll(resp.Body)
 	panic(fmt.Errorf("unexpected HTTP status code (%d): %s", resp.StatusCode, string(errorBytes)))
 }
+
+func TestCacheCR(t *testing.T) {
+	spec := tutils.DefaultSpec(testKube)
+	name := strcase.ToKebab(t.Name())
+	spec.Name = name
+	testKube.CreateInfinispan(spec, tutils.Namespace)
```

Comment:
Infinispan CR need to be removed after test. That's the reason why CO can fails with limiting resources and running Pods from the finished tests

Reply:
@durgeshanaokar You can do this with `defer testKube.DeleteInfinispan(spec, tutils.SinglePodTimeout)`

Reply:
@ryanemerson and @dmvolod can you please review the changes against the comment.